cnd/src/config.rs

```diff
@@ -180,11 +221,11 @@ fn assert_lnd_url_https(lnd_url: Url) -> Result<Url> {
     }
 }
 
-impl Default for Lightning {
-    fn default() -> Self {
+impl Lightning {
```

Comment:
I am not sure having two `impl` blocks really make sense.

Reply:
It doesn't! :D

Left over from amending a lot :grimacing: